stix_shifter_modules/datadog/stix_transmission/api_client.py

```diff
@@ -62,10 +62,10 @@ def get_processes_results(self):
         configuration = datadog_api_client.v2.Configuration(host=self.connection["site_url"])
         configuration.api_key["apiKeyAuth"] = self.auth["api_key"]
         configuration.api_key["appKeyAuth"] = self.auth["application_key"]
-        if self.connection["selfSignedCert"]:
+        if "selfSignedCert" in self.connection:
```

Comment:
you already verified and set the `selfSignedCert` in __init__() why are you doing it again?

Reply:
@mdazam1942  
Since there is difference in configuration object. In **init()** method, we have used `datadog_api_client.v1.Configuration` to create configuration object (for events dialect). And here we have to use `datadog_api_client.v2.Configuration` to create conf object for processes dialect. 

By default, we are considering `events` dialect but if user fires command with `processes` dialect then we need to change configuration object.